generate.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
 
-while getopts g:t:m:u:w: option
+while getopts g:t:d:m:u:w: option
 do
  case "${option}"
  in
  g) GITURL=${OPTARG};;
  t) export DOCTHEME=${OPTARG};;
+ d) export DEBUG=${OPTARG};;
```

Comment:
since I'm also not completely sure, if we come across some weird behavior with concurrent execution, I would directly pass all configuration vars as parameters to `config.py`. That way we would be able to remove export if required. sounds cool?